xbmc/platform/win32/WIN32Util.h

```diff
@@ -110,4 +111,26 @@ class CWIN32Util
    * \return Formatted string
    */
   static std::string FormatHRESULT(HRESULT hr);
+
+private:
+#ifndef TARGET_WINDOWS_STORE
+  struct DisplayConfigSnapshot
+  {
+    std::vector<DISPLAYCONFIG_PATH_INFO> paths;
+    std::vector<DISPLAYCONFIG_MODE_INFO> modes;
```

Comment:
modes member is not used !!!!!

I mean set, but never read...

Reply:
yes, it's possible to return a vector of DisplayConfigId (it's all we need with current callers) instead, but I'm afraid this could make some future extensions more complicated, as this simplification would have to be undone. I'll keep as separate commit that won't be sqashed as a compromise.

Reply:
Maybe return only `std::vector<DISPLAYCONFIG_PATH_INFO> paths` instead of the struct. This would simplify the code quite a bit.

Reply:
actually not true, GetDisplayTargetId() requires access to the path sourceInfo, not only the targetInfo.

Reply:
> Maybe return only `std::vector<DISPLAYCONFIG_PATH_INFO> paths` instead of the struct. This would simplify the code quite a bit.

That's what I ended up doing. doesn't really simplify much, but there is nothing unused.